z3c/dependencychecker/package.py

```diff
@@ -176,3 +177,21 @@ def _get_sources_top_level(self):
             os.sep,
         )
         sys.exit(1)
+
+
+class Package(object):
+
+    def __init__(self, path):
```

Comment:
Can you make "path" a bit more explicit? Absolute base path to the root of the package, I guess?